src/Components/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './NoResults.scss';
+
+type Props = {
+  category: string
+};
+
+export const NoResults: React.FC<Props> = ({ category }) => (
```

Comment:
```suggestion
export const emptyState: React.FC<Props> = ({ category }) => (
```

Reply:
it is supposed to be called NoResults in the task, isn't it?